e2e/package.json

```diff
@@ -9,7 +9,7 @@
     "test": "cross-env WORKBENCH_ENV=dev jest --maxWorkers=3",
     "test:debug": "cross-env WORKBENCH_ENV=dev HEADLESS=false jest --detectOpenHandles",
     "test-local": "cross-env WORKBENCH_ENV=local jest --maxWorkers=2",
-    "test:ci": "cross-env NODE_ENV=development CI=true jest --ci --maxWorkers=2 --forceExit",
+    "test:ci": "cross-env NODE_ENV=development CI=true jest --ci --maxWorkers=1",
```

Comment:
Unrelated to CircleCI config change, but it's related to running e2e tests in CircleCI. Abort job early if more than 3 tests have failed.